### ISSUE TITLE
Gittyup does not see directories

### ISSUE THREAD

**Issue by TheMegaDriver** [Murmele/Gittyup#638]
We have our repos stored under /local/ but Gittyup does not show this directory and cannot open it. I am the owner of the directory.

Replicate
Create a git clone of any repo you wnat under /local/
Try to add it as existing repo

Expected
/local/ shows in the file picker

Actual
/local/ cannot be selected, no even typed into the dialogue.

System
Ubuntu 20 LTS

**Comment by Murmele** [comment]
Did you use the flatpak version or? There only home directories are allowed. You can try with [Flatseal](https://flathub.org/apps/com.github.tchx84.Flatseal) to add additional permission to the /local folder

**Comment by TheMegaDriver** [comment]
It works now, but only if I clone the repo with Gittyup. For existing repos I can add them but everytime I wand to do anything I get an Permission denied error

**Comment by Murmele** [comment]
But if you clone with Gittyup are they at the correct location?

**Comment by TheMegaDriver** [comment]
Yes. It works fine on the correct location on the file system if I clone it
with Gittyup. If I add it as an existing directory I can still see
everything but every action just results in a failure due to the
permissions.

Murmele ***@***.***> schrieb am Di. 13. Feb. 2024 um 17:01:

**Comment by AHSauge** [comment]
Potentially fixed by #1000
